warehouse/forms.py

```diff
@@ -10,13 +10,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import annotations
+
+import typing as t
```

Comment:
Why the alias?

Reply:
It's a common way to shorten the overall line length. If this is the sticking point, I'll revert it, but I've got a long branch on type hints that makes heavy use of it.

Reply:
I may also change my stance later, after reviewing current practices a bit more, but that'll delay getting this out.

Reply:
Removed alias